configBase.js

```diff
@@ -29,7 +29,17 @@ module.exports = {
                             emit(doc.$content.samples[i]);
                         }
                     }
-                }
+                },
+                designDoc: 'analysis'
+            },
+            analysisBySampleAndId: {
```

Comment:
not sure about this name. why not just `analysisById` ?

Reply:
because that's what you emit: the id of the analysis

Reply:
but in there there's the uuid of the sample, that's why I though that this name would said more

Reply:
I agree with Michael that `analysisById` seems better. The key you are emitting is the `$id` of the analysis.